docs/src/doc/docs/user_guide/base-specific/frontend.md

```diff
@@ -81,22 +81,24 @@ To customize HTML output, you can use the [default template](https://github.com/
     To change page assets, you can set properties `customAssets` and `customStyleSheets`.
     Assets are handled by Dokka itself, not FreeMaker.
 
-Currently, there is only one template file with predefined name `base.ftl`. It defines general design of all pages to render.  
-If `templatesDir` is defined, Dokka will find the `base.ftl` file there.
+There is a template file with predefined name `base.ftl`. It defines general design of all pages to render.
+`base.ftl` can import another templates that can be set by user as well.
```

Comment:
I assume the user can overwrite individual templates, like only the footer? 

Can you please add which templates can be overridden and what it's used for? i.e
```
* footer.ftl - overrides the footer portion of the page, which includes the footer message and the copyright
```

Reply:
> I assume the user can overwrite individual templates, like only the footer?
Yes.

> Can you please add which templates can be overridden and what it's used for? i.e

I suppose the description of templates is too verbose. 
A file path should be enough. WDYT?

Reply:
Yeah, if it's easy to deduce meaning from the name, perhaps it's too much, you're right :+1: 

I'd say the only one which might raise questions is `article.ftl` (if it doesn't get renamed) - maybe it's worth to mention that it contains the main content of the page.

Anyhow, it's up to you -- I think listing all templates by name/path is the main thing